lms/djangoapps/program_enrollments/models.py

```diff
@@ -14,7 +14,7 @@
 from model_utils.models import TimeStampedModel
 from opaque_keys.edx.django.models import CourseKeyField
 from simple_history.models import HistoricalRecords
-from student.models import CourseEnrollment as StudentCourseEnrollment
+from student.models import AlreadyEnrolledError, CourseEnrollment as StudentCourseEnrollment
```

Comment:
It's weird to me to alias `CourseEnrollment` as `StudentCourseEnrollment` - we use the `CourseEnrollment` model all over this repo, there's a pretty solid understanding of what it means, why not leave it as `CourseEnrollment`?

Reply:
yeah I'm not seeing any great reason for that.  I'll make the change

Reply:
I don't see any good reason it's this way.  fixed.